### PR TITLE
Update Schema Automatically

### PR DESCRIPTION
**Pull request by tdddblog**
(1) See https://github.com/NASA-PDS/pds-registry-app/issues/146
(2) Java docs

**Test Instructions**
- Delete old registry: ```regfistry-manager delete-registry```
- Create new registry: ```regfistry-manager create-registry```
- Harvest some OREX data, for example ```https://sbnarchive.psi.edu/pds4/orex/orex.ovirs/data_raw/orbit_r/20191117T030457S365_ovr_scil0.xml```
- Load harvested OREX data: ```registry-manager load-data ...```
- Registry manager should automatically download OREX LDD and load harvested OREX data.

### CODE REVIEW

**Review by tloubrieu-jpl — CHANGES_REQUESTED** [review]
Hi @tdddblog ,

I was not able to make the test procedure you describe work:

S./bin/registry-manager load-data -dir ~/tmp/son_pere                                  less /Users/loubrieu/tmp/sElasticsearch URL: http://localhost:9200
            Index: registry

Updating schema with fields from /Users/loubrieu/tmp/son_pere/fields.txt
[ERROR] Could not find datatype for field 'orex:Time/orex:start_sclk'.
See 'https://nasa-pds.github.io/pds-registry-app/operate/common-ops.html#Load' for more information.

I am using this harvest configuration 
[bundles.xml.zip](https://github.com/NASA-PDS/pds-registry-mgr-elastic/files/6474547/bundles.xml.zip)

File: src/main/java/gov/nasa/pds/registry/mgr/dd/parser/AttributeDictionaryParser.java [hunk 1]
```diff
@@ -2,13 +2,28 @@
...
 import java.io.File;
 import com.google.gson.stream.JsonToken;
-import gov.nasa.pds.registry.mgr.dd.parser.DDAttribute;
 
+import gov.nasa.pds.registry.mgr.util.Logger;
 
-public class AttributeDictionaryParser extends BaseDDParser
+/**
+ * PDS LDD JSON file parser. 
+ * Parses "dataDictionary" -> "attributeDictionary" subtree.
```
[the remark this answers is not in the document]
Comment (tdddblog) [reply src/main/java/gov/nasa/pds/registry/mgr/dd/parser/AttributeDictionaryParser.java]: Javadocs are fixed

**Comment by tdddblog** [comment]
To troubleshoot your issue with OREX schema I need all files from your Harvest output directory. Can you zip it and attach or email to me. Also can you please provide full log for "registry-manager load-data" command.

**Comment by tloubrieu-jpl** [comment]
> To troubleshoot your issue with OREX schema I need all files from your Harvest output directory. Can you zip it and attach or email to me. Also can you please provide full log for "registry-manager load-data" command.

That is my mistake, I used an older version of the registry-manager. This works with the current version

**Review by tloubrieu-jpl — APPROVED** [review]
mvn javadoc:jar works now

**Pull request merged**